src/main/java/se/sundsvall/invoicesender/integration/db/entity/ItemType.java

```diff
@@ -0,0 +1,7 @@
+package se.sundsvall.invoicesender.integration.db.entity;
+
+public enum ItemType {
+	UNKNOWN,
```

Comment:
Out of curiosity, what the difference between UNKNOWN and OTHER?

Reply:
When an item is first persisted, the type is unknown. 

At a later stage we check the type for each item. In that check we either change the type to INVOICE or OTHER. At this stage, the type is known and hence no longer unknown.